Update UG to include '0' values for height/weight (#310)
AY2324S2-CS2103T-T17-3/tp: docs/UserGuide.md

Format: `add n/NAME p/PHONE_NUMBER [e/EMAIL] [a/ADDRESS] [w/WEIGHT] [h/HEIGHT] [nt/NOTE] [t/TAG]…​`

> * When HEIGHT and/or WEIGHT is specified to be 0, no weight/height will be added to the client.

Reviewer: ```suggestion
* When HEIGHT and/or WEIGHT is specified to be 0, no height/weight will be added to the client.
```

> * When HEIGHT and/or WEIGHT is specified to be 0, the values will not be added to the client's details.

Reviewer: ```suggestion
* When HEIGHT and/or WEIGHT is specified to be 0, no weight/height will be added to the client.
```

> * When HEIGHT and/or WEIGHT is specified to be 0, the respective height/weight values will not be shown on the client's details.

Reviewer 2: Would it be better to mention that it will not be added to the client?

> * When HEIGHT and/or WEIGHT is specified to be 0, the respective height/weight values will not be shown on the client's details.

Author: ```suggestion
* When HEIGHT and/or WEIGHT is specified to be 0, the values will not be added to the client's details.
```

> * When HEIGHT and/or WEIGHT is specified to be 0, the respective height/weight values will not be shown on the client's details.

Reviewer: ```suggestion
* When HEIGHT and/or WEIGHT is specified to be 0, no height/weight will be added to the client.
```

* Existing values will be updated to the input values.
* When no value is specified after a prefix, the value of that prefix will be removed from the client. (e.g. `edit 1 nt/` removes the note from the client at index 1).
    * Since WEIGHT can store multiple historical values, specifying an empty weight prefix removes the latest weight value from the client. Otherwise, the latest weight value will be replaced by the specified weight in this command.

> * When WEIGHT is specified to be 0, the latest weight (if any) will be removed from the client.

Author: ```suggestion
* When WEIGHT is specified to be 0, the **latest** weight (if any) will be removed from the client.
```
